src/logic/contracts/safeContracts.ts

```diff
@@ -134,16 +140,16 @@ export const instantiateSafeContracts = async () => {
   const networkId = await getNetworkIdFrom(web3)
 
   // Create ProxyFactory Master Copy
-  proxyFactoryMaster = getProxyFactoryContractInstance(web3, networkId)
+  proxyFactoryMaster = getProxyFactoryContractInstance(web3, networkId.toString() as ETHEREUM_NETWORK)
```

Comment:
It's possible to do `as ETHEREUM_NETWORK` once where the variable is defined.

Is it guaranteed that the network id from web3 will be defined in ETHEREUM_NETWORK?